public/app/core/components/ForgottenPassword/ChangePassword.tsx

```diff
@@ -1,28 +1,33 @@
 import React, { SyntheticEvent } from 'react';
 
 import { selectors } from '@grafana/e2e-selectors';
-import { Tooltip, Form, Field, VerticalGroup, Button } from '@grafana/ui';
+import { Tooltip, Form, Field, VerticalGroup, Button, Alert } from '@grafana/ui';
 
 import { submitButton } from '../Login/LoginForm';
 import { PasswordField } from '../PasswordField/PasswordField';
 interface Props {
   onSubmit: (pw: string) => void;
   onSkip?: (event?: SyntheticEvent) => void;
+  showDefaultPasswordWarning?: boolean;
 }
 
 interface PasswordDTO {
   newPassword: string;
   confirmNew: string;
 }
 
-export const ChangePassword = ({ onSubmit, onSkip }: Props) => {
+export const ChangePassword = ({ onSubmit, onSkip, showDefaultPasswordWarning }: Props) => {
   const submit = (passwords: PasswordDTO) => {
     onSubmit(passwords.newPassword);
   };
   return (
     <Form onSubmit={submit}>
       {({ errors, register, getValues }) => (
         <>
+          <h2>Update your password</h2>
+          {showDefaultPasswordWarning && (
+            <Alert severity="info" title="Continuing to use the default password exposes you to security risks." />
```

Comment:
Maybe this should be a warning?

Reply:
didn't wanna make it a warning since you haven't really done anything _wrong_ yet, but i could be convinced either way tbh 😄